app/jobs/health/flag_ineligible_patients_job.rb

```diff
@@ -0,0 +1,71 @@
+module Health
+  class FlagIneligiblePatientsJob < ActiveJob::Base
+    def perform(inquiry)
+      response = inquiry.eligibility_response
+
+      if response
+        # Update the last check date
```

Comment:
Should this be in a transaction?  Is it bad if it only partially finishes?  I'm guessing the clearing should probably only happen if it finishes.